body.hbs

```diff
@@ -0,0 +1,40 @@
+<section id="abstract">
+  <p>
+    This is the most recent interoperability report for Vaccine Credentials using the <a href="https://w3c-ccg.github.io/vaccination-vocab/">Vaccine Certificate Vocabulary</a>.
+  </p>
+</section>
+<section id="sotd">
```

Comment:
We need to summarize the number of vax certificates, tests, passes, and fails in the Status of the Document section in a way that's hard to miss and easily scannable/readable.

Reply:
![vaccine-report-5](https://user-images.githubusercontent.com/278280/115461727-bfb16480-a1f7-11eb-92a1-16fe1ba04f08.png)